pkg/daemon/ceph/client/config.go

```diff
@@ -293,7 +293,7 @@ func WriteCephConfig(context *clusterd.Context, clusterInfo *ClusterInfo) error
 	if err != nil {
 		return errors.Wrap(err, "failed to copy connection config to /etc/ceph. failed to read the connection config")
 	}
-	err = ioutil.WriteFile(DefaultConfigFilePath(), src, 0444)
+	err = ioutil.WriteFile(DefaultConfigFilePath(), src, 0600)
```

Comment:
Just to check to give this update permission to ceph.conf doesn't harm or create any secruity problem